CHANGELOG.md

```diff
@@ -30,6 +31,39 @@
 - [1.5.0](#150)
 - [1.0.0](#100)
 
+# 2.6.1
+
+This release mainly bugfix and improves test cases for `Manager API`.
+
+`Manager API` 2.6.1 should be used with [Apache APISIX 2.6](https://apisix.apache.org/downloads/). It is not recommended to use with other Apache APISIX versions.
+
+### Core
+
+- Feat: support HTTPS for Manager API [#1824](https://github.com/apache/apisix-dashboard/pull/1824)
+- Feat: run manager-api as an OS agnostic service [#1667](https://github.com/apache/apisix-dashboard/pull/1667)
+- Feat: refactor Plugin Orchestration [#1813](https://github.com/apache/apisix-dashboard/pull/1813)
+- Feat: add the service page upstream select option [#1633](https://github.com/apache/apisix-dashboard/pull/1633)
+- Feat: improve the Duplicate Route feature [#1833](https://github.com/apache/apisix-dashboard/pull/1833)
```

Comment:
Actually, it's not a good practice when you release a patched version but carrying so many features.